src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -171,6 +171,8 @@ public String deploy(AppDeploymentRequest request) {
 				.then(() -> operations.applications()
 					.start(StartApplicationRequest.builder()
 						.name(name)
+						.stagingTimeout(deploymentProperties.getStagingTimeout())
```

Comment:
There are `stagingTimeout` and `startupTimeout` options for `PushApplicationRequest` as well. Not sure if they are different from the ones updated in `StartApplicationRequest`?